src/plugins/vis_types/vislib/public/vislib/components/tooltip/_tooltip.scss

```diff
@@ -1,12 +1,15 @@
 .visTooltip,
 .visTooltip__sizingClone {
-  @include euiToolTipStyle('s');
```

Comment:
@elastic/kibana-visualizations I'm not sure where this tooltip is being used or how to reproduce it/see it in the browser - I'm fairly sure this change should work without major issue, but please let me know if not.

Reply:
Hi @cee-chen you can test it by turning on the `visualization:visualize:legacyHeatmapChartsLibrary` advanced setting and creating an Agg Based heatmap.

I've tested locally and I believe we need to bring all the properties there, because the color you have assigned are different then before.
Before:
<img width="389" alt="Screenshot 2024-10-02 at 10 08 14" src="https://github.com/user-attachments/assets/ca92825b-549d-4f60-a174-25864da6a2f7">
After:
<img width="609" alt="Screenshot 2024-10-02 at 10 03 48" src="https://github.com/user-attachments/assets/0359fd75-7727-44d8-b63a-9b0cfdeceab7">

Reply:
Unfortunately the code there is not React so you can't use the hooks for EUI.
Probably just copying the shadow/border/background/color is enough

Reply:
Ah I hadn't realized the difference was that stark, I'll copy the shadeOrTint Sass exactly in that case. It's still fairly fragile however as a warning - If EuiToolTip's styles ever change in the future (there is talks of a new theme on the horizon) then you'll need to manually update your custom tooltip to match.

Reply:
Definitely, but just consider that these tooltip and visualization editor will be remove in the future, so don't worry about that too much

Reply:
@cee-chen I'm just waiting for this to approve, whenever you have it ready please ping me